tests/pw/pages/productsPage.ts

```
@@ -622,6 +747,854 @@
+        await this.saveProduct();
+        await this.toHaveValue(productsVendor.minMax.minimumQuantity, '0');
+        await this.toHaveValue(productsVendor.minMax.maximumQuantity, '0');
+    }
+
+    // product edit
+
+    async editProduct(product: product['simple']): Promise<void> {
+        await this.goToProductEdit(product.editProduct);
+        await this.clearAndType(productsVendor.price, product.regularPrice());
+        await this.saveProduct();
+    }
```

Biome

[error] Duplicate class member name "editProduct"



(lint/suspicious/noDuplicateClassMembers)